sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Generated/Samples/Samples_ConversationAnalysisClient.cs

```diff
@@ -193,72 +222,159 @@ public async Task Example_AnalyzeConversation_AllParameters_Async()
 
         [Test]
         [Ignore("Only validating compilation of examples")]
-        public void Example_GetAnalyzeConversationJobStatus_ShortVersion()
+        public void Example_Conversations_AnalyzeConversation_AllParameters_Convenience()
         {
-            Uri endpoint = new Uri("<endpoint>");
+            Uri endpoint = new Uri("<https://my-service.azure.com>");
             AzureKeyCredential credential = new AzureKeyCredential("<key>");
             ConversationAnalysisClient client = new ConversationAnalysisClient(endpoint, credential);
 
-            Response response = client.GetAnalyzeConversationJobStatus(Guid.Parse("73f411fe-4f43-4b4b-9cbd-6828d8f4cf9a"));
+            AnalyzeConversationInput analyzeConversationInput = new ConversationalInput(new ConversationAnalysisInput(new TextConversationItem("<id>", "<participantId>", "<text>")
+            {
+                Language = "<language>",
+                Modality = InputModality.Transcript,
+                Role = ParticipantRole.Customer,
+            }), new ConversationActionContent("<projectName>", "<deploymentName>")
+            {
+                Verbose = true,
+                IsLoggingEnabled = true,
+                StringIndexType = StringIndexType.TextElementsV8,
+                DirectTarget = "<directTarget>",
+                TargetProjectParameters =
+{
+["key"] = new LuisConfig
+{
+Query = "<query>",
+CallingOptions = new LuisCallingConfig
+{
+Verbose = true,
+Log = true,
+ShowAllIntents = true,
+TimezoneOffset = 1234,
+SpellCheck = true,
+BingSpellCheckSubscriptionKey = "<bing-spell-check-subscription-key>",
+},
+ApiVersion = "<apiVersion>",
+}
+},
+            });
+            Response<AnalyzeConversationActionResult> response = client.AnalyzeConversation(analyzeConversationInput);
+        }
+
+        [Test]
+        [Ignore("Only validating compilation of examples")]
+        public async Task Example_Conversations_AnalyzeConversation_AllParameters_Convenience_Async()
+        {
+            Uri endpoint = new Uri("<https://my-service.azure.com>");
+            AzureKeyCredential credential = new AzureKeyCredential("<key>");
+            ConversationAnalysisClient client = new ConversationAnalysisClient(endpoint, credential);
+
+            AnalyzeConversationInput analyzeConversationInput = new ConversationalInput(new ConversationAnalysisInput(new TextConversationItem("<id>", "<participantId>", "<text>")
+            {
+                Language = "<language>",
+                Modality = InputModality.Transcript,
+                Role = ParticipantRole.Customer,
+            }), new ConversationActionContent("<projectName>", "<deploymentName>")
+            {
+                Verbose = true,
+                IsLoggingEnabled = true,
+                StringIndexType = StringIndexType.TextElementsV8,
+                DirectTarget = "<directTarget>",
+                TargetProjectParameters =
+{
+["key"] = new LuisConfig
```

Comment:
SHould we remove references to LuisConfig?

Reply:
It is still there not deprecated yet.